From 9ca0da024b536f8e18468f70af0b82e9dd4994d0 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Fri, 14 Sep 2018 12:18:02 +0300
Subject: [PATCH] hl-fw-loader: improve versioning

Remove the hardcoded meaningless version.
Provide the version from the git repository by version-hooks
package means.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/package/hl-fw-loader/hl-fw-loader.mk b/package/hl-fw-loader/hl-fw-loader.mk
index 638555b..e09b1ba 100644
--- a/package/hl-fw-loader/hl-fw-loader.mk
+++ b/package/hl-fw-loader/hl-fw-loader.mk
@@ -1,4 +1,3 @@
-HL_FW_LOADER_VERSION = 0.1.0
 HL_FW_LOADER_SITE = hl-fw-loader
 HL_FW_LOADER_LICENSE = unclear
 
diff --git a/package/version-hooks/version-hooks.mk b/package/version-hooks/version-hooks.mk
index 5ffa47a..4451a21 100644
--- a/package/version-hooks/version-hooks.mk
+++ b/package/version-hooks/version-hooks.mk
@@ -12,3 +12,5 @@
 UBOOT_POST_RSYNC_HOOKS += HL_VERSION_GEN
 LINUX_POST_RSYNC_HOOKS += HL_VERSION_GEN
 
+HOST_HL_FW_LOADER_POST_RSYNC_HOOKS += HL_VERSION_GEN
+
